tests/mocha/clinical_browser.js

```diff
@@ -229,5 +235,15 @@ function showDataInVariantBrowser(driver){
     });
 }
 
+function clinVarReset(driver){
+    clinVarSearchByLocation(driver);
+    driver.findElement(By.xpath("//div[contains(@id,'ClinvarWidgetPanel')]//span[text()='Reset']")).click();
+    driver.wait(until.elementLocated(By.xpath("//div[contains(@id,'clinvar-browser-grid-body')]//table[1]//td[1]/div[text()]")), 30000).then(function(text) {
+        driver.findElement(By.xpath("//div[contains(@id,'clinvar-browser-grid-body')]//table[1]//td[1]/div[text()]")).getText().then(function(text){
+            chai.assert.equal(text, '17');
```

Comment:
Is this "17" the chromosome number or am I misunderstanding the test? The same question applies to the function in `variant_browser.js`.